nasa/harmony
# Harmony 1282 batching

## Jira Issue ID
HARMONY-1282

## Description
Adds batching for aggregation services that require it. This allows us to process a large number of files with aggregation services without exceeding system or service limits.

## Local Test Steps
1. add the following to your .env file
CMR_MAX_PAGE_SIZE=2
MAX_BATCH_INPUTS=3
2. Restart Harmony then execute the following query
http://localhost:3000/C1234208438-POCLOUD/ogc-api-coverages/1.0.0/collections/all/coverage/rangeset?maxResults=10&concatenate=true
3. Verify that multiple `concise` work items are created. 

## PR Acceptance Checklist
